src/User.php

```diff
@@ -35,20 +35,21 @@ public function getUsername(): string
 
     public function getAvatar(): string
     {
-        return $this->avatar;
+        return "./assets/imgs/avatars/" . $this->avatar;
     }
```

Comment:
it might be good, because this is no longer a simple getter, to rename the method to `getFullAvatarPath` so it's clear it's doing more than just getting. it will also need a unit test